### PR TITLE
Partiprogrammer and skolelove instances included

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Added partiprogrammer & skolelove
Little cleanup in facets, both js & css
Made sure we actually handle the default template for instancs (Shame on whoever just put in a comment about handling it!)
Minor-minor adjustment for frontpage.

### CODE REVIEW

File: src/components/searchresult/addons/ResultImage.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}),
+
+methods: {
+  showFullPicture() {
+    this.fullPicture = !this.fullPicture
+  }
+},
+
+mounted() {
+},
```
Comment (jorntx) [inline src/components/searchresult/addons/ResultImage.vue:30]: Unused code.

Comment (jesperlauridsen) [reply src/components/searchresult/addons/ResultImage.vue]: Removed.

File: src/containers/InstanceContainer.vue [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
     <div v-if="instanceName">
       <div class="titleContainer">
-        <router-link :to="{name: 'Home'}"><h2>Meloar</h2></router-link>
+        <h2>{{ instanceName || 'Ukendt instans' }}</h2>
         <hr style="width:300px" />
-        <span>{{ instanceName || 'Ukendt instans' }} v2</span>
+        <router-link :to="{name: 'Home'}"><span>back to MELOAR v2</span></router-link>
       </div>
       <search-box :time="searchOptions.time"/>
       <search-map v-if="searchOptions.map === true" />
```
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/containers/InstanceContainer.vue]: Fixed.

File: src/components/SearchResults.vue [hunk 3]
```diff
@@ -75,8 +75,12 @@ export default {
       solrOptions: state => state.searchStore.solrOptions,
     }),
      searchResultComponentName() {
+       let instanceComponent = false
+       Object.keys(this.$options.components).forEach(item => {
+        item.includes(this.instance) && item.includes("SingleSearchResult") ? instanceComponent = true : null
+       })
           //Remember to handle default instance
-         return this.instance ? 'SingleSearchResult_' + this.instance : 'SingleSearchResult_default'
+         return instanceComponent === false ? 'SingleSearchResult_default' : 'SingleSearchResult_' + this.instance
```
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/components/SearchResults.vue]: Fixed.

**Pull request merged**